index.rst

```diff
@@ -19,7 +19,7 @@ In this documentation
    .. grid-item-card:: :ref:`Start here <start-here>`
        :columns: 12
 
-       Get started with Ubuntu Pro: :ref:`Set up your account <account-setup>`, :ref:`attach a machine to your subscription <get_token_and_attach>` and learn :ref:`how to use the Pro client <tutorial-commands>`
+       Get started with Ubuntu Pro: :ref:`Set up your account <account-setup>`, :ref:`attach a machine to your subscription <attach>` and learn :ref:`how to use the Pro client <tutorial-commands>`
```

Comment:
@s-makin it should be "attach" or "how_to_attach"?
<Ignore if my comment doesnt make sense at all>

Reply:
In this case it's `attach` because we're using the `:ref:` role, which points to the label/anchor inside the `how_to_attach.rst` file (the anchor is denoted with the `.. _attach:` formatting). It's possible to use the filename, but for that we'd want to use the :doc: role instead of :ref: to tell Sphinx that we're pointing it to a document/file and not an anchor inside a file :)

![image](https://github.com/user-attachments/assets/a908e704-2f59-43c8-9dad-3f48c191fd55)